libpdbg/device.c

```diff
@@ -48,11 +48,118 @@ static const char *take_name(const char *name)
 	return name;
 }
 
+static const char *get_unitname(const struct pdbg_target *node)
```

Comment:
do not move the whole function rather if required you can do forward declaration of just the function, or else too many changes to review.

Reply:
Sure